keps/sig-node/3619-supplemental-groups-policy/README.md

```diff
@@ -919,6 +987,16 @@ For GA, this section is required: approvers should be able to confirm the
 previous answers based on experience in the field.
 -->
 
+
+A pod with `supplementalGroupsPolicy: Strict` may be rejected by kubelet with the probablility of $$B/A$$,
```

Comment:
Only if the user fails to target the pod via a nodeSelector, right?

Reply:
Yes, if the cluster administrator maintains node labels propagated from `node.status.features.supplementalGroupsPolicy`.